config/initializers/statsd.rb

```diff
@@ -25,7 +25,7 @@
 
 %w[success failure].each do |s|
   StatsD.increment(V0::SessionsController::STATSD_SSO_CALLBACK_KEY, 0, tags: ["status:#{s}", 'context:unknown'])
```

Comment:
I think we'd need to update this initializer to have all of the tags from `saml_callback_tags`.  It looks like we might possibly need to initialize each possible combo of values to 0.